examples/src/instantiation.js

```diff
@@ -61,67 +61,75 @@ export const profileJson = {
       {
         width: 1800,
         height: 1712,
-        verificationFunction: 'keccak256(bytes)',
-        verificationData:
-          '0xfbcfcbbc86d886e862419361c48251d778884a90429d36c8d002559cbcb52972',
+        verification: {
+          method: 'keccak256(bytes)',
```

Comment:
Why here do we use `keccak256(bytes)`?

While in the previous file `example/src/getData.js` we use `keccak256(utf8)`?

![image](https://github.com/ERC725Alliance/erc725.js/assets/31145285/46fa5933-092a-4c37-9899-aa28ff64256f)

Not sure as both are related to a LSP3Profile. I guess one of the example might be wrong.

Reply:
I think it's because for LSP3Profile it refers to the utf8 JSON data.

Reply:
The keccak256(bytes) is used for images for example.

Reply:
Double checked this and that's correct 👍 

![image](https://github.com/ERC725Alliance/erc725.js/assets/31145285/d248a25f-bf4c-462e-9915-6750548837e1)

![image](https://github.com/ERC725Alliance/erc725.js/assets/31145285/6abf2db4-d1db-49da-870d-f3aef6001c4b)